purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/events/CustomerCenterDisplayMode.kt

```diff
@@ -1,16 +1,14 @@
 package com.revenuecat.purchases.customercenter.events
 
 import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI
-import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 
 /**
  * Display mode for the Customer Center. Meant for RevenueCatUI use.
  */
 @Serializable
 @ExperimentalPreviewRevenueCatPurchasesAPI
-enum class CustomerCenterDisplayMode {
+enum class CustomerCenterDisplayMode(val value: String) {
```

Comment:
Hmm any reason to have it as a property instead of the serialized name?

Reply:
I saw that `PaywallEventType` was doing the same thing. Is there a clean way to access the `Serialized` value?

Reply:
Not really, other than serializing it again... But ideally, our code doesn't need the serialized value except to send it to the backend/store it in the file/retrieve it from the file I guess?

Reply:
It's used here
```kotlin
    return BackendStoredEvent.CustomerCenter(
        BackendEvent.CustomerCenter(
            id = creationData.id.toString(),
            revisionID = data.revisionID,
            type = data.type.value,
            appUserID = appUserID,
            appSessionID = data.sessionIdentifier.toString(),
            timestamp = data.timestamp.time,
            darkMode = data.darkMode,
            locale = data.locale,
            isSandbox = data.isSandbox,
            displayMode = data.displayMode.value,
        ),
    )
```

Is there a way to access the serialized value like `.rawValue`? 🤔

Reply:
I guess I could encode it / decode it instead. Let me try that out

Reply:
Fixed, can you please take another look @tonidero